### PR TITLE
Activity log: use activity endpoint

### PR DESCRIPTION
**Pull request by sirreal**
Add `QueryActivityLog` component.
Use `activity` endpoint.
Cleans up timestamp logic and language.

To test:
1. Visit `http://calypso.localhost:3000/stats/activity/$site` for a Pressable site with activity.
1. Check network requests and watch for traffic at the `activity` endpoint.
1. Ensure network traffic looks OK

Followup:
* `logError` reducer
* reducer tests

Replaces #14795

### CODE REVIEW

File: client/state/data-layer/wpcom/activity-log/activity/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	activityLogError,
+	activityLogUpdate,
+} from 'state/activity-log/actions';
+// import addQueryArgs from 'lib/route/add-query-args';
+
+const activityLogRequest = ( { dispatch }, action ) => {
+	dispatch( http( {
+		apiVersion: '1',
+		method: 'GET',
+		path: `/sites/${ action.siteId }/activity`,
```
Comment (seear) [inline client/state/data-layer/wpcom/activity-log/activity/index.js]: The convention of the folder structure of data layer modules is to match the endpoint URL, so we should move this up to `/wpcom/activity` I think.

**Comment by seear** [comment]
The new test is failing, seems like some suspect `site_id` handling:

```
site_id: 1,
...
site_id: 2,
```

File: client/my-sites/stats/activity-log/index.jsx [hunk 2]
```diff
@@ -373,7 +383,12 @@ export default connect(
 			rewindStatusError: getRewindStatusError( state, siteId ),
 
 			// FIXME: Testing only
-			isPressable: get( state.activityLog.rewindStatus, [ siteId, 'isPressable' ], false ),
+			isPressable: true,
```
Comment (sirreal) [inline client/my-sites/stats/activity-log/index.jsx]: This change needs to be reverted, it was for testing only.

File: client/my-sites/stats/activity-log/index.jsx [hunk 3]
```diff
@@ -373,7 +383,12 @@ export default connect(
...
+			logs: get( state, [
```
Comment (sirreal) [inline client/my-sites/stats/activity-log/index.jsx]: If we have a `logItems` selector, it should be used. If not, we can follow-up in another PR.

File: client/state/data-layer/wpcom/activity/index.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/**
```
Comment (sirreal) [inline client/state/data-layer/wpcom/activity/index.js]: To follow convention and reflect the endpoint route, this should probably be moved under `sites`.

**Pull request merged**